src-java/floodlight-service/floodlight-modules/README.md

```diff
@@ -15,3 +15,32 @@ where:
  ratePkts - is the rate in packets for specific type of rule
  disco_packet_size - the size of discovery packet, is being defined in properties file with the key 'org.openkilda.floodlight.switchmanager.SwitchManager.disco-packet-size'.
 ```
+
+#### How to debug Floodlight component
+
+In order to run Floodlight in the debug mode, we have to add the following arguments
+`"-agentlib:jdwp=transport=dt_socket,address=50506,suspend=n,server=y”` 
+when running the app.
+Particularly in Floodlight we can not do this straight to the Dockerfile since it leads us to entrypoint.sh script.
+We have to add the arguments above to the
+`projectroot/docker/floodlight-modules/entrypoint.sh` file.
+To do so, in this file find `exec java` line under the ```"$1" = 'floodlight’```  condition block.
+In this line add arguments for debug and leave the existing arguments as is.
+As result the exec java line should look something like this:
```

Comment:
`As a result, ...`
If you decided to use `exec java` as the inline code block on line 27, then please use it in the same manner on line 29 (and other places)

Reply:
fixed